package.json

```diff
@@ -53,5 +54,10 @@
       "post-rewrite": "node bin/post-rewrite.js",
       "pre-push": "yarn test-all"
     }
+  },
+  "dependencies": {
+    "@koa/router": "^8.0.1",
+    "convict": "^5.1.0",
```

Comment:
convict is used by firefox send to manage options. It's handy because it automatically fetches options from environment variables, with default values suitable for the dev environment.

Reply:
Great! Does it give good error messages then if the environment variables are malformed? This seems to be a common source of bugs and deployment issues.

Reply:
Yeah it should; but then we'll need to put stricter rules maybe, like custom validation functions. Currently a lot are just "string"...